dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -190,7 +190,7 @@ private static Path generate(Bom bom) throws IOException, TemplateException, URI
     List<ClassPathEntry> artifactJarsInBom = classpath.subList(0, managedDependencies.size());
     ImmutableSet<ClassPathEntry> entryPoints = ImmutableSet.copyOf(artifactJarsInBom);
 
-    LinkageChecker linkageChecker = LinkageChecker.create(classpath, entryPoints);
+    LinkageChecker linkageChecker = LinkageChecker.create(classpath, entryPoints, null);
```

Comment:
Madd `LinkageChecker.create(classPath)` public; the dashboard does not use reachability analysis or exclusion files.